src/java.base/linux/native/libsimdsort/avx512-32bit-qsort.hpp

```diff
@@ -0,0 +1,443 @@
+/*
+ * Copyright (c) 2021, 2023, Intel Corporation. All rights reserved.
+ * Copyright (c) 2021 Serge Sans Paille. All rights reserved.
+ * Intel x86-simd-sort source code.
```

Comment:
I don't think this line here and in other new files will pass our Copyright header checker. Do you need this line here? You do have comment below `This implementation is based on x86-simd-sort`.
`DO NOT ALTER ..` line should immediately follow `Copyright` lines.

Reply:
Fixed the copyright headers. Please see the latest commit that was pushed.